cmd/kubeapps-apis/buf.gen.yaml

```diff
@@ -8,6 +8,22 @@ plugins:
     out: gen
     opt:
       - module=github.com/kubeapps/kubeapps/cmd/kubeapps-apis/gen
+      # The following option controls whether the generated interfaces require
+      # the `mustEmbedUnimplemented**Server` method. This method is useful to
+      # ensure that when you add a method to a proto service and regenerate
+      # files, they will automatically implement the new method and return
+      # "Unimplemented", as opposed to not compiling until the new method is
+      # added explicitly. But a plugin can still embed the
+      # `Unimplemented**Server` struct, it's just not enforced. The
+      # unintentional downside of enforcing it via the
+      # `mustEmbedUnimplemented**Server` for us is that the
+      # `mustEmbedUnimplemented**Server` method is not exposed, which means that
+      # *only* implementations in the same package (ie. with access to the same
+      # unexposed method) can satisfy the interface. We explicitly want plugins
+      # defined in other packages to be able to satisfy a generic gRPC server
+      # interface.
+      # TODO: create upstream issue about this.
```

Comment:
I'll add an upstream issue and link here on Monday.

Reply:
So, just for clarification: there is nothing wrong in continue using `require_unimplemented_servers=true` if the `mustEmbedUnimplemented**Server` was exposed when generating the code.
So, the upstream issue is for them to expose it, am I getting it right?

Reply:
Woops, still need to create the upstream issue. Yes, that's what I'll be suggesting upstream, though there may be a reason I'm not aware of as to why they want it private.